pkgs/development/python-modules/dm-sonnet/default.nix

```diff
@@ -19,6 +20,8 @@ let
 
   # first build all binaries and generate setup.py using bazel
   bazel-build = buildBazelPackage {
+    bazel = bazel_0_26;
```

Comment:
Why are you making these changes? They appear unrelated / unnecessary to the upgrade of Bazel?

Reply:
Ah, I think I see your justification above in the description.